config/notifiers.go

```diff
@@ -275,11 +275,14 @@ func (c *WebhookConfig) UnmarshalYAML(unmarshal func(interface{}) error) error {
 type OpsGenieConfig struct {
 	NotifierConfig `yaml:",inline"`
 
-	APIKey      Secret            `yaml:"api_key"`
-	APIHost     string            `yaml:"api_host"`
-	Description string            `yaml:"description"`
-	Source      string            `yaml:"source"`
-	Details     map[string]string `yaml:"details"`
+	APIKey       Secret            `yaml:"api_key"`
+	APIHost      string            `yaml:"api_host"`
+	Description  string            `yaml:"description"`
+	Source       string            `yaml:"source"`
+	Details      map[string]string `yaml:"details"`
+	Teams        []string          `yaml:"teams"`
+	Tags         []string          `yaml:"tags"`
+	LabelsToTags []string          `yaml:"labels_to_tags"`
```

Comment:
Yes, only `LabelsToTags`.

If say semicolon is forbidden in Opsgenie tags you could have a single string like `a;b;c;d` that's then split out. That'd permit the full power of templating.